src/dat.c

```diff
@@ -144,10 +153,7 @@ int eraDat(int iy, int im, int id, double fd, double *deltat )
    enum { NERA1 = (int) (sizeof drift / sizeof (double) / 2) };
 
 /* Dates and Delta(AT)s */
-   static const struct {
-      int iyear, month;
-      double delat;
-   } changes[] = {
+   static const eraLEAPSECOND _changes[] = {
```

Comment:
This only uses the globally defined `eraLEAPSECOND` structure, defined in `erfa.h`.